Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
-
+gem "rake"
```

Comment:
Doesn't look like Rakefile is in this repo, but you may want to add the `github-pages` gem like in https://github.com/canada-ca/ore-ero/blob/master/Gemfile